### DISCUSSION TITLE
Inquiry about Energy Label Source in DeepMD-kit's Fitting Network

### DISCUSSION THREAD

**Discussion by yanjin-tech** [deepmodeling/dpdata#652] [Q&A]
Hello,

I have a question regarding the fitting network within the DeepMD-kit framework. The model predicts the energy of each central atom based on the local environment descriptors of each atom, and then these energies are summed to obtain the total energy. However, **where do the labels for the energy of the central atoms come from**? When I use **dpdata** to read the structure, I **only get the total energy of the structure**.

Could you please clarify how the individual atomic energy labels are derived or provided within DeepMD-kit?

Thank you for your assistance.

Best regards,

**Comment by njzjz** [comment]
The atomic energy has no physical meaning, so cannot be labeled.

**Reply by yanjin-tech** [reply]
Hello，

I aim to train a model using the DeepMD-kit framework to predict the Atomic Displacement Parameter (ADP). The ADP is very helpful for studying the non-harmonic effects, thermal expansion properties, and thermoelectric properties of crystals. Therefore, developing a predictive model for ADP is of significant importance. The ADP shares two similarities with the central atom energy in the machine-learned interatomic interaction potential:

The ADP is also a scalar, corresponding to one value per atom.
Similar to the central atom energy, the ADP is influenced by the local environment of the central atom within the cutoff radius.
Based on this, I hope to adapt and modify the DeepMD-kit architecture to change the target of the fitting network from the energy, force, and stress tensor of the central atom to the ADP of the central atom, i.e., to establish a mapping between atomic configurations and ADP, thereby enabling the prediction of ADP values for each atom within unseen structures.

My questions are as follows:

If the single energy does not have physical meaning, could you please advise on how I might proceed to change the target of the fitting network from the central atom's energy to the ADP?

Thank you for your guidance.

**Reply by njzjz** [reply]
See https://github.com/deepmodeling/deepmd-kit/issues/3040

**Reply by yanjin-tech** [reply]
Hello again,

Thank you very much for your valuable insights and assistance，it has been extremely helpful to me in advancing my research. I wish you continued success in your work.
